### PR TITLE
feat(web): handle different registry namespace

### PR DESCRIPTION
**Pull request by peterambrus3**

### CODE REVIEW

File: protobuf/proto/crux.proto [hunk 1]
```diff
@@ -453,6 +457,14 @@ enum RegistryType {
   GOOGLE = 5;
 }
 
+enum RegistryNamespace {
+  NAMESPACE_UNKNOWN = 0;
```
Comment (m8vago) [inline protobuf/proto/crux.proto:461]: RNS_UNKNOWN,
RNS_ORGANIZATION,
etc.

**Review by balanceee — APPROVED** [review]
LGTM

File: web/crux/prisma/schema.prisma [hunk 2]
```diff
@@ -308,6 +309,14 @@ enum RegistryTypeEnum {
   google
 }
 
+enum RegistryNamesapaceEnum {
+  unknown
```
Comment (m8vago) [inline web/crux/prisma/schema.prisma:317]: is unknown ever happens?

File: web/crux/src/app/registry/registry.mapper.ts [hunk 3]
```diff
@@ -82,7 +85,7 @@ export default class RegistryMapper {
 
   detailsToDb(
     request: CreateRegistryRequest | UpdateRegistryRequest,
-  ): Pick<Registry, 'url' | 'type' | 'apiUrl' | 'user' | 'token' | 'imageNamePrefix'> {
+  ): Pick<Registry, 'url' | 'type' | 'apiUrl' | 'user' | 'token' | 'imageNamePrefix' | 'namespace'> {
```
Comment (m8vago) [inline web/crux/src/app/registry/registry.mapper.ts:88]: would be nice to have a type for this pick declared. at least locally

File: web/crux/src/app/registry/registry.mapper.ts [hunk 4]
```diff
@@ -93,6 +96,7 @@ export default class RegistryMapper {
         apiUrl: null,
         token: null,
         user: null,
+        namespace: RegistryNamesapaceEnum.unknown,
```
Comment (m8vago) [inline web/crux/src/app/registry/registry.mapper.ts:99]: why not just null?

**Review by m8vago — CHANGES_REQUESTED** [review]

File: web/crux/prisma/schema.prisma (hunk 2 above)
Comment (peterambrus3) [reply web/crux/prisma/schema.prisma:317]: yes in case of v2, dockerhub, and google

File: web/crux/prisma/schema.prisma [hunk 5]
```diff
@@ -70,18 +70,19 @@ model Node {
...
+  updatedAt       DateTime?              @default(now()) @db.Timestamptz(6)
+  updatedBy       String?                @db.Uuid
+  name            String                 @db.VarChar(70)
   description     String?
   icon            String?
   url             String
   user            String?
   token           String?
   type            RegistryTypeEnum
+  namespace       RegistryNamesapaceEnum @default(unknown)
```
Comment (polaroi8d) [inline web/crux/prisma/schema.prisma:85]: Typoooo `RegistryNamesapaceEnum`

File: web/crux/src/app/registry/registry.mapper.ts (hunk 4 above)
Comment (polaroi8d) [reply web/crux/src/app/registry/registry.mapper.ts:99]: I think it's okay. We talked with Peti about this, and came to the conclusion unknown is much better.

**Review by polaroi8d — CHANGES_REQUESTED** [review]

File: web/crux/prisma/schema.prisma (hunk 5 above)
Comment (peterambrus3) [reply web/crux/prisma/schema.prisma:85]: too late, its known :|

File: web/crux/src/app/registry/registry.mapper.ts (hunk 4 above)
Comment (peterambrus3) [reply web/crux/src/app/registry/registry.mapper.ts:99]: we talked this with @polaroi8d in person, but i can make the field nullable if necessary

File: web/crux/prisma/schema.prisma (hunk 5 above)
Comment (polaroi8d) [reply web/crux/prisma/schema.prisma:85]: What do you mean?

Comment (peterambrus3) [reply web/crux/prisma/schema.prisma:85]: nvm :D we talked this earlier with @robot9706 in person, I'm working on it

**Review by robot9706 — APPROVED** [review]
LGTM 😎

**Review by m8vago — APPROVED** [review]

**Review by polaroi8d — APPROVED** [review]

**Pull request merged**